modules/auxiliary/dos/tcp/claymore_dos.py

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env python
+# Note, works with both python 2.7 and 3
+
+
+import socket
+import json
+
+from metasploit import module
+
+metadata = {
+    'name': 'Claymore Dual GPU Miner  Format String dos attack',
+
+    'description': '''
+    Claymore’s Dual GPU Miner 10.5 and below is vulnerable to a format strings vulnerability. This allows an 
+    unauthenticated attacker to read memory addresses, or immediately terminate the mining process causing 
+    a denial of service.
+    ''',
+
+    'authors': [
+        'res1n',  # Vulnerability disclosure
+        'bluebird',  # Metasploit external module (Python)
+    ],
+
+    'date': '2018-02-06',
+
+    'references': [
+        {'type': 'cve', 'ref': 'CVE-2018-6317'},
+        {'type': 'url', 'ref': 'https://www.exploit-db.com/exploits/43972/'},
```

Comment:
Unless there's something different with python vs ruby modules, this can be type 'edb' and ref '43972'

Reply:
That should work